### DISCUSSION TITLE
Welcome to Swordfish Discussions!

### DISCUSSION THREAD

**Discussion by rmraya** [rmraya/Swordfish#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by githubber** [comment]
Thank you for this! 
How do you recommend updating? Im a few versions behind now... do I just pull, or is there a different git command?
Thank you.

**Comment by rmraya** [comment]
Simply clone again.
